Backpack/src/main/java/net/skyscanner/backpack/starrating/BpkInteractiveStarRating.kt

```diff
@@ -38,7 +38,10 @@ open class BpkInteractiveStarRating @JvmOverloads constructor(
     val x = if (layoutDirection == View.LAYOUT_DIRECTION_RTL) width - ev.x else ev.x
     val itemWidth = width / maxRating
     val selectedItems = x / itemWidth
-    rating = Math.max(1f, selectedItems + 0.5f)
+    super.rating = Math.round(Math.max(1f, selectedItems + 0.5f)).toFloat()
```

Comment:
Why are we adding `Math.round` here?

Reply:
Interactive Star Rating supports only Integer values because that's what user sees on the screen. You cannot have half-star on interactive rating. Besides without rounding the callback will be receiving something like 3.33241, 3.7534 instead of 3 and 4.